### PR TITLE
Show first time boot status in oled

### PR DESCRIPTION
**Pull request by srepac**

### CODE REVIEW

File: scripts/kvmd-bootconfig [hunk 1]
```diff
@@ -82,6 +89,10 @@ if [ -n "$FIRSTBOOT$FIRST_BOOT" ]; then
 		unset disk part npart label
 	fi
 
+    ### create NFS_Primary directory for use with NFS MSD after reboot ###
```
Comment (mdevaev) [inline scripts/kvmd-bootconfig:92]: This is not related with the current PR, pelase remove this part

File: scripts/kvmd-bootconfig [hunk 2]
```diff
@@ -48,6 +48,13 @@ rw
 # ========== First boot configuration ==========
 
 if [ -n "$FIRSTBOOT$FIRST_BOOT" ]; then
+    ### stop regular kvmd-oled service and show first time setup status in oled
+    systemctl stop kvmd-oled
+    /usr/bin/kvmd-oled --interval=0 --text="First time setup\nDo NOT interrupt\nPlease Wait...\n"
+    
+    ### force reboot after first boot
```
Comment (mdevaev) [inline scripts/kvmd-bootconfig:55]: Why?

File: scripts/kvmd-bootconfig [hunk 3]
```diff
@@ -48,6 +48,13 @@ rw
 # ========== First boot configuration ==========
 
 if [ -n "$FIRSTBOOT$FIRST_BOOT" ]; then
+    ### stop regular kvmd-oled service and show first time setup status in oled
+    systemctl stop kvmd-oled
```
Comment (mdevaev) [inline scripts/kvmd-bootconfig:52]: Why we need to stop it here?

File: scripts/kvmd-bootconfig (hunk 2 above)
Comment (srepac) [reply scripts/kvmd-bootconfig:55]: If you don't temporarily stop kvmd-oled service, the message will get clobbered.

File: scripts/kvmd-bootconfig (hunk 1 above)
Comment (srepac) [reply scripts/kvmd-bootconfig:92]: ok

File: scripts/kvmd-bootconfig (hunk 3 above)
Comment (srepac) [reply scripts/kvmd-bootconfig:52]: If you don't temporarily stop kvmd-oled service, the message will get clobbered.

File: scripts/kvmd-bootconfig (hunk 2 above)
Comment (mdevaev) [reply scripts/kvmd-bootconfig:55]: I mean why you need a reboot here?

File: scripts/kvmd-bootconfig (hunk 3 above)
Comment (mdevaev) [reply scripts/kvmd-bootconfig:52]: Got it
